src/store/ClientStore.js

```diff
@@ -56,9 +56,27 @@ const ClientStore = types.model('ClientStore', {
       }
   }),
 
+  get: flow(function* get (request) {
+    return yield auth.checkBearerToken().then((token) => {
+      self.setBearerToken(token)
+      return self.tove.get(request)
+    })
+  }),
+
+  patch: flow(function* get (request, body) {
+    return yield auth.checkBearerToken().then((token) => {
+      self.setBearerToken(token)
+      return self.tove.patch(request, body)
+    })
+  }),
+
   setBearerToken: (token) => {
-    self.tove.jwt(token)
-    self.toveZip.jwt(token)
+    // If Token is null, we need to remove jwt tokens
+    // If token is a string with a length, we need to set tokens
+    if (token === null || token.length) {
+      self.tove.jwt(token)
+      self.toveZip.jwt(token)
+    }
```

Comment:
One comment based on the frisbee docs:

> api.jwt(token) - helper function that sets a JWT Bearer header. It accepts the jwt_token as a single string argument. If you simply invoke the function null as the argument for your token, it will remove JWT headers.

If the token isn't defined, perhaps this should call `self.tove.jwt()` and `self.toveZip.jwt()`? I'm not sure if there is any other handling this app should have, but the header likely should be removed if it had been set.